workers/tcp.go

```diff
@@ -0,0 +1,24 @@
+package workers
+
+import (
+	"log"
+	"net"
+)
+
+func InitializeTCPHealthCheck(l *net.TCPListener) {
+	go tcpHealthCheckListener(l)
+}
+
+func tcpHealthCheckListener(listener *net.TCPListener) {
+	for {
+		conn, err := listener.Accept()
+
+		if err != nil {
+			log.Fatal("not able to accept connection: %s", err)
```

Comment:
yep, the healthcheck from the loadbalancer relies on this being present, so if we can't listen/talk on this port, then the service is useless.